ui/app/AppLayouts/Chat/components/StickerButton.qml

```diff
@@ -242,8 +242,9 @@ Item {
             id: mouseArea
             anchors.fill: parent
             acceptedButtons: Qt.LeftButton | Qt.RightButton
-            cursorShape: Qt.PointingHandCursor
+            cursorShape: root.isPending ? Qt.ArrowCursor : Qt.PointingHandCursor
```

Comment:
Maybe `enabled: false`, or `enabled: !root.isPending` in your case, would do both the checks at the same time